blog/2023-03-iso-certification/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "ISO certification"
+path: "/blog/2023-03-iso-certification"
+date: "2023-03-03T17:46:00"
+author: minajanicijevic
+---
+
+## Triply Takes Security to the Next Level with ISO Certification
+
+We are excited to announce that Triply has received ISO 27001:2017 certification, a globally recognized standard that ensures organizations have implemented effective and robust information security management systems. This certification highlights our commitment to excellence and dedication to providing our customers with the highest quality and most secure products and services.
+
+ISO 27001:2017 is an international standard that specifies the requirements for establishing, implementing, maintaining, and continually improving an information security management system (ISMS) within an organization. This standard provides a systematic approach to managing sensitive company information, including intellectual property, financial information, and employee data.
+
+So, what does this certification mean for our customers? It means that they can trust us to take security seriously and to continually improve our security practices. It's a testament to our organization's ability to control information security risks and maintain the confidentiality, integrity, and availability of our customers' information.
+
+But how did we achieve this certification? It all started two years ago when we established a security management system. This system formalized our existing processes and implemented a mindset change among management and staff to become more security-aware. In November 2021, our certification was granted by Brand Compliance, and in November 2022, we were subsequently renewed, with compliments acknowledging the maturity of our organization recognizing the maturity of our organization. To this day, we continue to prioritize our security practices with the help of our information security coordinator, who ensures that our security work is maintained at the highest level possible.
```

Comment:
This sentence: `(...) with compliments acknowledging the maturity of our organization recognizing the maturity of our organization.` seems to have a duplicate phrase.